src/Uno.Toolkit.UI/Controls/AutoLayout/AutoLayout.Layouting.Arrange.cs

```diff
@@ -47,6 +47,11 @@ protected override Size ArrangeOverride(Size finalSize)
 			var child = children[i];
 			var calculatedChild = _calculatedChildren[i];
 
+			if (child is FrameworkElement { Visibility: Visibility.Collapsed })
```

Comment:
@Robert-Louis  there should have been a runtime time for this change. Was it possible to create one?